pkg/logs/diagnostic/message_receiver.go

```diff
@@ -126,14 +133,25 @@ func shouldHandleMessage(m *message.Message, filters *Filters) bool {
 	return shouldHandle
 }
 
-func formatMessage(m *message.Message) string {
-	return fmt.Sprintf("Name: %s | Type: %s | Status: %s | Timestamp: %s | Service: %s | Source: %s | Tags: %s | Message: %s\n",
+func formatMessage(m *message.Message, redactedMsg []byte) string {
+	hostname, err := util.GetHostname()
+	if err != nil {
+		hostname = "unknown"
+	}
+
+	ts := time.Now().UTC()
+	if !m.Timestamp.IsZero() {
+		ts = m.Timestamp
+	}
+
+	return fmt.Sprintf("Integration Name: %s | Type: %s | Status: %s | Timestamp: %s | Hostname: %s | Service: %s | Source: %s | Tags: %s | Message: %s\n",
 		m.Origin.LogSource.Name,
 		m.Origin.LogSource.Config.Type,
-		m.GetStatus(),
+		ts,
 		m.Timestamp,
```

Comment:
Why do we use the timestamp for the status ?

Reply:
oops - I think this got mixed up. Good catch.